components/web/ejabberd/Makefile

```diff
@@ -23,36 +23,36 @@
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
 # Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
 # Copyright 2022 Niklas Poslovski
+# Copyright 2024 Goetz Fischer, R-A-C. All rights reserved.
 #
 
 BUILD_BITS= 64
-
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ejabberd
 COMPONENT_FMRI= web/server/$(COMPONENT_NAME)
-COMPONENT_VERSION= 22.0.5
-HUMAN_VERSION= 22.05
-COMPONENT_PROJECT_URL=	http://www.ejabberd.im/
-COMPONENT_SRC= $(COMPONENT_NAME)-$(HUMAN_VERSION)
+COMPONENT_VERSION= 23.10
+HUMAN_VERSION= 23.10
+COMPONENT_PROJECT_URL= http://www.ejabberd.im/
```

Comment:
Please switch to https here.

Reply:
http://www.ejabberd.im/ is just a redirect to https://www.ejabberd.im/ so yes, seriously.  What is wrong with https://www.ejabberd.im/ ?

Reply:
```suggestion
COMPONENT_PROJECT_URL= https://www.ejabberd.im/
```